pkg/inject/uninject.go

```diff
@@ -58,6 +74,18 @@ func (conf *ResourceConfig) uninjectPodSpec(report *Report) {
 	t.Volumes = volumes
 }
 
+func uninjectNamespace(ns *v1.Namespace, report *Report) {
```

Comment:
Hey, Can we directly use the `uninjectObjectMeta` ,method and just pass in `ns.ObjectMeta`, Any Problem doing that way?

Because, the already defined function, even removes the proxy config labels and also helps us re-use code.

Reply:
+1 For reusing the uninjectObjectMeta function if possible

Reply:
@Pothulapati @zaharidichev  thanks for your suggestion! Initially, I was under the impression that we'd necessarily have to typecast to `v1.Namespace` to get the annotations from `conf`. I was able to simply pass the namespace `conf` as `con thanks for your suggestion! Initially, I was under the impression that we'd necessarily have to typecast to `v1.Namespace` to get the annotations from `conf`. I was able to simply pass the namespace `conf` as `conf.workload.Meta` (i.e without having to perform typecasting) to `uninjectObjectMeta()` and carry out the uninject process as expected! This also helped me reduce the number of lines of code